### PR TITLE
Remove Step 4 from Math 2 #27320

### PR DESCRIPTION
**Pull request by kushaldabbe**

### CODE REVIEW

File: files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md [hunk 1]
```diff
@@ -43,7 +43,6 @@ Try updating the live code below to recreate the finished example by following t
 1. Multiply `result` and `result2` and assign the result back to `result` (use assignment shorthand).
 2. Format `result` so that it has two decimal places and store it in a variable called `finalResult`.
 3. Check the data type of `finalResult` using `typeof`. If it's a `string`, convert it to a `number` type and store the result in a variable called `finalNumber`.
-4. Update the original calculation so that they give you a value of `10.42`. Don't change the numbers or the operators.
 
 To pass this test, `finalNumber` should have a result of `10.42`.
```
Comment (Josh-Cena) [inline files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:47]: You would need to change this too :)

**Comment by wbamberg** [comment]
Thanks for your PR! Apart from @Josh-Cena 's comment, we also need updates to https://github.com/mdn/learning-area/, specifically (I think):

https://github.com/mdn/learning-area/blob/3c3c3babeb030f3919a76d7f6783197cd8407b14/javascript/introduction-to-js-1/tasks/math/math2-download.html#L29 and https://github.com/mdn/learning-area/blob/3c3c3babeb030f3919a76d7f6783197cd8407b14/javascript/introduction-to-js-1/tasks/math/math2.html#L28.

File: files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md (hunk 1 above)
Comment (kushal752) [reply files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:47]: Can you please be a little specific? What should I change?

Comment (Josh-Cena) [reply files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:47]: After you deleted the 4th step, the final result is no longer 10.42, so you would need to figure out what the new result should be.

Comment (kushal752) [reply files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md:47]: Considering the BODMAS rule should I change the 10.42 to 4633.33?

**Comment by kushal752** [comment]
@Josh-Cena Please have a look now and see If any more changes need to be done.

**Comment by Josh-Cena** [comment]
Content should be good; could you send another PR for learning-area?

**Comment by kushal752** [comment]
> Content should be good; could you send another PR for learning-area?

Sure, I'll have a look at it.
This content PR should be good to merge then right?

**Comment by Josh-Cena** [comment]
Yes, although I'll let @wbamberg handle it since he's more familiar with the learning area :)

**Comment by kushal752** [comment]
@wbamberg
Please have a look at both the PRs from content and learning-area for merging and see if any more changes need to be done.

**Comment by kushal752** [comment]
@wbamberg @Josh-Cena 
Can you please have a look at both these PRs ASAP? 
I haven't received any response from @wbamberg.

**Comment by Josh-Cena** [comment]
No need to repeatedly ping. Reviewers have priorities and will get to yours when they have time. Our PR turnover rate is usually 1w–1mo, so you shouldn't be worried unless the staleness goes beyond that.

**Review by hamishwillee — APPROVED** [review]
Thanks very much @kushal752 - good fix! PS The linked PR also good.

**Pull request merged**